library/core/src/alloc/layout.rs

```diff
@@ -225,14 +225,11 @@ impl Layout {
     /// padding required to get a 4-aligned address (assuming that the
     /// corresponding memory block starts at a 4-aligned address).
     ///
-    /// The return value of this function has no meaning if `align` is
-    /// not a power-of-two.
```

Comment:
`align` is a parameter and this is a safe method, so I don't think removing this text is acceptable.

Personally I'd like to see this method omitted from this stabilization pass, so we could consider changing the type to something that doesn't have the "well you'll just get garbage if you're not careful" behaviour.

Reply:
Ah, I misinterpreted the comment since I was thinking too much on the wrong side of the interface about it. I've reverted this part of the stabilization.